### PR TITLE
Add light microscopy parameters

### PR DESCRIPTION
**Pull request by jenniferboedker**
describe the light microscopy DTO with related properties

### CODE REVIEW

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/lightmicroscopy/LightMicroscopyParameters.groovy [hunk 1]
```diff
@@ -1,12 +1,107 @@
...
+     * Power is measured in milliwats (mW)
+     */
+    private final Power laserPowerBeamSplitter
+
+    /**
+     * Time required to generate an image with the camera
+     *
+     * In the range of femtoseconds to picoseconds
+     */
+    private final Time cameraAcquisitionTime
```
Comment (sven1103) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/lightmicroscopy/LightMicroscopyParameters.groovy:62]: Maybe we go for the Java-native `Date` object here
```suggestion
    private final Time cameraAcquisitionTime
```

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Area.groovy [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package life.qbic.datamodel.dtos.imaging.properties
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
```
Comment (sven1103) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Area.groovy:6]: Detailed description is missing somehow

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Position.groovy [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package life.qbic.datamodel.dtos.imaging.properties
+
+/**
+ * <class short description - One Line!>
```
Comment (sven1103) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Position.groovy:4]: Description is missing

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Power.groovy [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package life.qbic.datamodel.dtos.imaging.properties
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
```
Comment (sven1103) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Power.groovy:6]: Description is missing

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Time.groovy [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package life.qbic.datamodel.dtos.imaging.properties
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
```
Comment (sven1103) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Time.groovy:7]: Maybe not needed anymore, see my previous comment

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Unit.groovy [hunk 6]
```diff
@@ -0,0 +1,13 @@
+package life.qbic.datamodel.dtos.imaging.properties
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
```
Comment (sven1103) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Unit.groovy:6]: Unsure if we need it. We need to check with @luiskuhn , what meaningful ranges and units are for a position.

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/lightmicroscopy/LightMicroscopyParameters.groovy (hunk 1 above)
Comment (jenniferboedker) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/lightmicroscopy/LightMicroscopyParameters.groovy]: Good Idea, a accepted your suggestion. For me a fixed unit would be fine. Maybe we need to check with @luiskuhn to make sure this is a good unit to choose?

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Area.groovy (hunk 2 above)
Comment (jenniferboedker) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Area.groovy:6]: aaec7b77

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Position.groovy (hunk 3 above)
Comment (jenniferboedker) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Position.groovy]: 0bde2a26

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Time.groovy (hunk 5 above)
Comment (jenniferboedker) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Time.groovy]: right, I removed it 👍

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Unit.groovy (hunk 6 above)
Comment (jenniferboedker) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Unit.groovy:6]: Good point, I also was a bit confused. Lets double check 👍

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Power.groovy (hunk 4 above)
Comment (jenniferboedker) [reply src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Power.groovy]: b2773f80

**Review by sven1103 — APPROVED** [review]
LGTM. the unit class can still be resolved later

**Pull request merged**